.github/workflows/pull-request.yml

```
@@ -295,7 +295,81 @@ jobs:
+  
+  Docker-Check:
+    needs: Test-Application
+    runs-on: ubuntu-latest
+         
+    steps: 
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
```

actionlint (1.7.3)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)